tests/commands/test_store_charmlibs.py

```diff
@@ -350,4 +376,114 @@ def test_getlibinternals_libid_bad_value(tmp_path, value, monkeypatch):
     )
 
 
+def test_getlibinternals_pydeps_complex(tmp_path, monkeypatch):
+    """The PYDEPS field can be multiline, unicode, different quotes."""
+    monkeypatch.chdir(tmp_path)
+    test_path = _create_lib(
+        pydeps="""PYDEPS = [
+        'foo',
+        "bar",
+        "moño",
```

Comment:
good thinking testing non-ascii text

Reply:
Yes! I don't remember any package with non-ascii chars but AFAIK they are not forbidden.